terraform/environments/delius-jitbit/onprem_gateway.tf

```diff
@@ -0,0 +1,110 @@
+# Pre-req - security group
+resource "aws_security_group" "onprem_gateway" {
+  name        = "onprem-gateway"
+  description = "Controls access onprem gateway instance"
+  vpc_id      = data.aws_vpc.shared.id
+  tags = merge(local.tags,
+    { Name = lower(format("OnPremiseDataGateway-%s-%s", local.application_name, local.environment)) }
```

Comment:
Feel free to ignore, but perhaps consider extracting this to a local variable and remove the lower function call

`on_prem_dgw_name = "OnPremiseDataGateway-${local.application_name}-${local.environment}"`